charts/tenant-base/chart/templates/deployment.yaml

```
@@ -1,28 +1,25 @@
 {{- range $name, $val := .Values.deployments }}
 {{- if .enabled }}
-
 ---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: {{ printf "%s-%s" (include "chart.fullname" $) $name }}
+  name: {{ $name }}
```

Could this be problematic? E.g. if I want to run a `mysql` pod for two different higher level applications?

---

The alternative would be?
```suggestion
  name: {{ printf "%s-%s" $.Release.Name $name }}
```

---

Yes, keeping the current behavior of including the `Release.Name` in the name should avoid the risk of naming conflicts.

---

Done